## 6.1: Discovering DNA
- ### 6.1.1: Molecular Genetics: the Protein vs. DNA Debate
	- [[Gregor Mendel]]: described factors that were responsible for passing traits on; didn't know what the factors were
	- coordinated scientific effort to identity genetic material = mid-1800s to mid-1900s
		- at beginning, researchers formalized qualities that the unknown genetic material must possess; 4 qualities:
			- 1. information
				- must store relatively large amt of info, enough to produce cell's [[proteins]] and other structures (bc DNA makes RNA makes proteins)
			- 2. replication
			- 3. communication
			- 4. change
		- **these 4 qualities make up the "brain" of the cell**
	- during 1800s and early 1900s, [[Friedrich Miescher]], [[Robert Feulgen]], and [[Archibald Edward Garrod]] contributed to understanding of the genetic material
		- [[Friedrich Miescher]]: extracted an acidic material from white blood cells' nuclei, calling it [[nucleic acid]]
			- #### type of [[nucleic acid]] found in chromosomes of all [[eukaryotic]] cells: [[DNA]]
			- later, was determined that [[nucleic acid]]s are made up of 4 [[nucleotide]]s: adenine, guanine, cytosine, and thymine
			- ![[Pasted image 20230816080801.png]]
		- [[Robert Feulgen]]: developed staining technique that could *quantify* DNA; the more DNA present in the nuclei, the darker the stain
		- ![[Pasted image 20230816081940.png]]
			- w/ this, researchers found that **DNA levels are species-specific**
			- across all species, sex cells consistently contained *half* the amount of DNA that the [[autosomal]]/unrelated to sex cells did
		- during this time, most scientists thought that [[protein]] (not [[DNA]]) was genetic material (even w/ evidence)

- ### 6.1.2: Continuing to Link Genes to Chemicals: Muller, Beadle, and Tatum
	- in 1920s: Hermann Muller and Thomas Hunt Morgan furthered genetics field
		- Muller: could increase mutation rate in fruit flies by subjecting them to x-rays
		- ![[Pasted image 20230816082422.png]]
			- 10C increase in temp = doubled mutation rate
	- George Beadle and Edward Tatum: subject red bread mold to x-rays, **greatly increasing mutation rate**
	- ![[Pasted image 20230816082431.png]]
		- normally mold can synthesize almost all of the organic compounds it needs + can be grown on [[minimal medium]] containing a few minerals, 1 vitamin, and an energy src
		- work:
			- 1. spores subjected to x-rays; placed on enriched media containing amino acids and allowed to grow until they produce spores that're collected
			- 2. spores crown on [[minimal media]]; if they can't, then a mutation has occurred bc of x-rays
			- 3. pieces of mutant mold = put in tubes containing minimal media +1 amino acid each; mutant strain can only grow when amino acid [[alanine]] present in tube
			- result indicated that mutation could be linked to defect in specific enzyme (genes + enzymes linked); "one gene one enzyme"

- ### 6.1.3: Griffith and Transformation
	- 1928: [[Frederick Griffith]] performed series of experiments w/ 2 strains of *Pneumococcus* bacteria
	- ![[Pasted image 20230816083107.png]]
		- smooth (S) strain: known to be [[virulent]] (deadly)
			- smooth colony type; mice die when injected
			- when heated/killed + injected, mice live (causes chemical change)
			- ![[Pasted image 20230816083059.png]]
		- rough (R) strain: [[avirulent]] (not deadly)
			- produced colonies w/ rough surface + type; mice live when injected
		- when heat-killed smooth *and* normal rough strai
		- ![[Pasted image 20230816083042.png]]ns are both injected, mice die
			- **conclusion: substance present in heat-killed smooth transformed living (rough) strain into smooth bacteria**
				- transforming substance: then thought to be genetic material, but not yet identified

- ### 6.1.4: Avery, MacLeod, and McCarty/Hershey and Chase: DNA Wins!
	- 1944: [[oswald avery]], [[colin macleod]], and [[maclyn mccarty]] showed that *nucleic acids isolated from heat-killed deadly (S) strain of a [[bacteriophage]]* responsible for *transforming harmless (R) strain deadly*
		- thus, **DNA, not protein, is genetic material**
	- 1952: [[alfred hershey]] and [[martha chase]]: series of experiments involving reproduction of [[bacteriophage]]s, (viruses that infect bacteria)
	- ![[Pasted image 20230816083554.png]]
		- hershey and chase capitalized on fact that *viruses are simple structures w/ 2 basic molecules: a protein coat and DNA*
			- one **had** to be genetic material, capable of using bacteria to produce more viruses
			- sulfur in [[bacteriophage]]: 35S; phosphorus in [[bacteriophage]]: 32P
		- during viral reproduction, protein coats *remained on the outside of the bacterial cells + could be separated w/o affecting viral replication*
			- new viruses produced in bacterial cells = radioactive DNA
		- **demonstrated that DNA is the genetic material**!!!

- ### 6.1.5: Chargaff and Franklin and Wilkins: The DNA Story Begins
	- DNA established as genetic material, so scientists insensified efforts to *characterize DNA's molecular structure*
		- structure must allow 4 basic functions of genetic material (see 6.1.1) which include storage of **large quantities of information**
			- achieved by 4 nucleotide subunits (illustrated below)
			- ![[Pasted image 20230816084015.png]]
				- 2 subuntis: [[purine bases]] (adenine and guanine)
				- other 2 subunits: [[pyrimidine bases]] (thymine and cytosine)
	- 1950: Erwin Chargaff found that **total amount of DNA: SPECIES-SPECIFIC**
		- within a given species amt of adenine relative to thymine = equal, as was amt of cytosine relative to guanine
	- Rosalind Franklin + Maurice Wilkins: used x-ray diffraction to study DNA structure
	- ![[Pasted image 20230816084340.png]]
		- technique involves striking DNA crystals w/ x-rays and photographing resulting diffraction pattern
		- **crisscross pattern provides evidence that DNA = helical in shape**
			- other scientists then used these measurements to furhter their understanding of DNA structure + function

## 6.2: DNA Structure Revealed
- ### 6.2.1: Watson and Crick: The Clues
	- [[James Watson]] and [[Francis Crick]] = first to *accurately* describe [[DNA]] structure
		- constructed hypothetical model that incorporated the Erwin Chargaff's findings + findings of [[Rosalind Franklin]] and [[Maurice Wilkins]]
			- 3 sets of data:
				- 1. chargaff's rules: erwin chargaff found that the amt of adenine and thymine are equal (and the amount of guanine and cytosine) are equal within a species
				- 2. repetitive amt ratios: franklin & wilkins' data showed that some element repeated every 0.34 nm and another repeated every 3.4 nm
				- ![[Pasted image 20230816123533.png]]
					- molecule ~2nm wide
					- prob ties into chargaff's rules
				- 3. shape of molecule: based on [[diffraction]] pattern, DNA molecule appeared helical in shape
			- based on 3 sets of data, proposed that **DNA was a double-stranded helix, each strand consisting of a series of [[nucleotide]]s**
				- data showed that the 2 strands = *exactly* 2 nm apart (a)
				- ![[Pasted image 20230816123911.png]]
				- (b) shows that each strand consists of nucleotide subunits of *a sugar, a phosphate, and a base*; these units are 0.34 nm apart
				- ![[Pasted image 20230816123927.png]]
					- sugar + phosphate form backbone of strand, and base available for hydrogen bonding with base on other strand
				- to remain consistent w/ width data, necessary for a [[pyrimidine]] (1-ring structure) to **always** pair w/ a [[purine]] (2-ring structure)
					- adenine (A) must bond *only* with thymine (T) and guanine (G) only with cytosine (C)
						- specific bonding between adenine and thymine + between guanine and cytosine *fits Chargaff's results!*
		- [[Linus Pauling]]: contemporary of watson + crick; proposed that DNA consisted of 3 nucleotide strands (incorrect; watson and crick's version proved to be correct)

- ### 6.2.2: Watson and Crick: The Double Helix
	- 4/25/1953: Watson and Crick's article "The Molecular Structure of Nucleic Acids: A Structure for DNA" appeared in journal *Nature*
		- understanding DNA structure enabled scientists to start untanglingg *process of how DNA serves as genetic material for cells*
	- DNA = described as a double helix bc composed of 2 [[nucleotide]] strands wound around each other
	- ![[Pasted image 20230817024531.png]]
		- (a): single nucleotide subunit
			- phosphate group, deoxyribose sugar, and 1 of 4 nitrogenous bases
			- carbons of sugar = numbers 1-5
			- each DNA strand made up of [[polynucleotide]] chain
		- (b): DNA molecule: double helix of polynucleotide chains
			- 2 strands = 2 nm apart
			- within strand, nucleotides .34 nm apart
			- 10 nucleotides in each turn of helix, so each turn occurs at 3.4 nm
			- repetition!
		- overall shape of molecular = comparable to twisted ladder
## 6.3: Introduction to DNA Replication
- ### 6.3.1: Replication: Meselson and Stahl
- 1958: Mathew Meselson and Franklin Stahl = experiments that tested the way  DNA replicates itself
	- bacteria = grown in heavy-nitrogen (15MN) medium for many generations
		cells then transferred to light-nitrogen (14N) medium and allowed to replicate for either one or two generations in light medium; process of *density gradient centrifugation* used 		to analyze weights of all 3 types of cells
	- Watson + Crick's model of double-stranded DNA molecule suggested that ***pair of nucleotide strands would separate and each would serve as template for new strand***
		- mechanism: [[semiconservative replication]]
	- competing w/ semiconservative model were *conservative* and *dispersive replication*
		- hypotheses of all 3: predicted diff weights for first/2nd gen cells
			cells subjected 2 one round of replication = bands intermediate in density, so DNA molecules consisted of 1 parental (heavy) and one new (light) strand
			therefore, conservative replication ruled out. cells subjected to 2 rounds = 1 intermediate and one light band, so dispersive replication ruled out
			density of cells in all 3 gens match pattern expected for *semi-conservative replication*
		- conservative: 2 parent strands wouldn't separate, replucation results in DNA molecular w/ 2 new strands
		- dispersive: parts of both parental strands separate + act as templates 4 new strand synthesis
- ### 6.3.2: DNA: Polymerization with Triphosphate Nucleotides
	- when a cell *divides* via [[semiconservative replication]], each new cell gets one original [[parental strand]] that serves as a template for the synthesis of a new [[complementary strand]]
		- this synthesis = DNA [[polymerization]]
			- [[polymerization]]: [[endergonic]] process, requiring expenditure of [[ATP]]
	- [[nucleotide triphosphate]]: contains energy in bonds that link 2nd and 3rd phosphate groups (which is then used to join the nucleotide to the [[growing strand]])
		- as part of new strand, [[nucleotide]] only has one phosphate group; nucleotides joined together by 3', 5' [[phosphodiester]] linkages
			- 3' and 5' designations refer to 3' and 5' carbons of the sugar molecule
			- ![[Pasted image 20230820131942.png]]
		- overall direction ([[orientation]]) of new synthesized strand = denoted by *its top and bottom nucleotides*, respectively
			- outer ends: not involved in phospho-diester linkage, resulting in empty 5' and 3' carbon
			- orientation of new strand said to be 5'-3' (orientation of [[complementary strand]] opposite of the [[parental strand]], meaning that they're [[antiparallel]])
			- ![[Pasted image 20230820132141.png]]


## 6.4: Events of DNA Replication
- ### 6.4.1: Events at the Replication Fork: The Leading Strand
	- [[prokaryotic]] cell = polymerizes 500 [[nucleotide]]s/sec
		- chromosome: circular, so replication occurs **efficiently**
	- [[eukaryotic]] cell = *synthesizes* 50 [[nucleotide]]s/sec
		- contains between 4-5 billion nucleotides(!)
		- use multiple [[replication bubble]]s, which greatly speed up replication
			- each [[replication bubble]] = two y-shaped [[replication fork]]s
	- DNA replication = *continuous* in one strand and *discontinuous* in the other
		- events basic to DNA process replication + specific to [[leading strand]]:
			- replication occurs at sites on molecule called [[replication bubbles]]; both strands replicated simultaneously at [[replication fork]]
				- ![[Pasted image 20230820132604.png]]
			- **overall events by which both strands are replicated**:
				- 1. strands separated and unwound by DNA [[helicase]]
				- 2. strands prevented from coming together by [[helix-destabilizing proteins]]
				- 3. enzymes that catalyze [[synthesis]] of new [[complementary strand]]s = [[DNA polymerases]]
					- can only add nucldeotides to the 3' end of [[growing strand]]
				- 4. [[RNA primer]] initiates synthesis of either new strand to provide first 3' end that enzyme requires
					- RNA primer = layer removed and substituted w/ DNA
				- synthesis of both complementary strands occurs in 5' to 3' direction
				- ![[Pasted image 20230820132955.png]]
					- strand growing towards replication fork grows continuously + called [[leading strand]] (which is [[antiparallel]] to the [[parental strand]] that served as its template)
- ### 6.4.2: Events at the Leading Strand, Part II
	- [[replication bubble]]: occurs at *origin of replication* and consists of 2 [[replication fork]]s
		- [[replication]] happens in opposite directions along each of replication forks (bubble enables DNA replication to occur in 2 directions)
			- [[DNA polymerases]] attaches to each replication fork
				- can **only** build new strands in the 5' to 3' direction (see img) so new strands synthesized in opposite directions
				- requires open 3' OH group to attach a new nucleotide; when just starting, that OH group is provided by 3' end of [[RNA primer]] strand that had been added by [[primase]]
					- first DNA nucleotide = added to open 3' OH group
					- nucleotides coming after = added to first DNA nucleotide
					- illustration shows new nucleotide that'll join strand; orientation of new strand opp. of orientation of [[parental strand]], so strands are [[antiparallel]]
						- ![[Pasted image 20230821121814.png]]
		- DNA strands = antiparallel (5' to 3' orientation runs in opp. directions)
			- [[leading strand]]: grows/synthesized continuously toward replication fork
			- [[lagging strand]]: grows/synthesized opp. of direction of replication fork
	- at each replication fork, og DNA strands = unwound (seen in img) in overall direction of unwinding and replication
		- img demonstrates [[antiparallel]] nature of [[DNA]] strands
		- ![[Pasted image 20230821115643.png]]
- ### 6.4.3: Events at the Replication Fork: The Lagging Strand
	- [[DNA polymerases]] add [[nucleotide]]s to the 3' end of a [[growing strand]], so only one of the new [[complementary strand]]s can grow continuously in the direction of the moving [[replication fork]]
		- [[leading strand]]: grows continuously
		- [[lagging strand]]: grows away from replication fork
	- the [[lagging strand]]:
		- discovery attributed to [[Reiji Okazaki]]
		- as the [[replication fork]] opens, [[leading strand]] comprises a continuous strand of new nucleotides growing in opening fork's direction
		- grows away from opening fork
			- ![[Pasted image 20230821123233.png]]
		- the two strands grow in opp. direction bc [[DNA polymerases]] can *only* add nucleotides to the 3' end of a strand
			- ![[Pasted image 20230821142926.png]]
			- as fork unwinds, [[lagging strand]] synthesized in 100-1000 nucleotide pieces called [[Okazaki fragment]]s
				- each [[Okazaki fragment]] = needs [[RNA primer]] to initiate synthesis
				- primers removed, [[DNA polymerases]] fills in gap w/ DNA nucleotides
				- fragments then joined together by [[ligase]]
		- *each of the [[parental strand]]s* serves as **template** for synthesis of a new [[complementary strand]]
		- #### end product of [[DNA replication]] is two *identical* DNA molecules
		- ![[Pasted image 20230821143108.png]]
			- question: is the final product 'flipped' bc of [[antiparallel]] strands?
- ### 6.4.4: Proofreading, End Replication, and [[Telomeres]]
	- DNA polymerase = synthesizes new nucleotides w/ great accuracy (ex. human genome is very, very complex, but roughly only *one nucleotide in a billion* is paired incorrectly)
	- [[DNA polymerases]]: responsible for [[genetic proofreading]]
		- works in conjunction w/ [[ligase]] to excise + replace incorrectly matched base pairs
	- DNA in a [[eukaryotic]] cell = rod-shaped, presenting a *problem* in DNA replication
		- [[telomeres]] come into play here
	- when growing [[lagging strand]] reaches end of molecule, last few DNA molecules can't be added
		- after RNA primer = removed, [[DNA polymerases]] can't fill gap w/ DNA bc DNA polymerase has no 3' OH end to attach to
		- DNA molecules shorten after each round of replication
			- ![[Pasted image 20230821143722.png]]
		- to avoid loss of genetic material, ends of DNA consist of long, repeating nucleotide strands called [[telomeres]] that don't code for cell products
		- [[telomeres]]: 
			- protect coding sequences of DNA
			- after many rounds of replication + shortening of the ends, noncoding sequences disappear
			- important coding sequences will be degraded
			- **certain cell lines possess enzyme called** ***telomerase*** **that can polymerize telomere sequences**
			- ![[Pasted image 20230822082414.png]]
				- cells that replicate many times, like [[germ cells]], have [[telomerase]]
			- typical repeated unit in human DNA = TTAGGG
			- ![[Pasted image 20230821144029.png]]

- ### 6.4.5: DNA Replication: A Summary
	- [[DNA]] consists of double helix of nucleotide strands (nucleotide subunit composed of sugar, phosphate, and either a purine or pyrimidine base)
		- within a strand, [[phosphodiester]] bonds join the nucleotides
			- sugars and phosphates form strand backbone
			- ![[Pasted image 20230822091134.png]]
		- between strands, specific [[base pairing]] occurs
		- ![[Pasted image 20230822091114.png]]
			- adenine and guanine = [[pyrimidine]]s (purines bond w/ pyrimidines)
			- adenine only bonds with thymine, and guanine bonds only with cytosine through hydrogen bonding
			- specificity = strands are *complementary* (see [[complementary strand]])
			- adenine, guanine, cytosine, and thymine are the nucleotides purine or pyrimidine base
	- initiation of replication happens when [[hydrogen]] bonds between strands break
	- ![[Pasted image 20230822090946.png]]
		- new strands synthesized in opp. directions of each other + called [[leading strand]] and [[lagging strand]]
		- molecule replicates via [[semiconservative replication]], w/ each strand acting as a template for the synthesis of a new [[complementary strand]]

## 6.5: Transcription
- ### 6.5.1: Transcription and Translation: An Overview
	- *central dogma of molecular genetics as formulated by* (Francis Crick):
		- **information flows from DNA to RNA to protein**.
		- no longer considered dogma; was given substance/built upon
	- transcription + translation: *describe* molecular processes by which info flows from DNA to RNA to protein (Crick's dogma)
		- [[transcription]]: enzymatic process by which genetic info in one strand of DNA is used to direct synthesis of a *complementary sequence of bases* of [[messenger RNA]] (mRNA)
			- during process, mRNA = like a scribe "transcribing" or making a new version of the master text (transcribes DNA info); new vers. then translated into protein
			- in prokaryotes, mRNA synthesized along [[template strand]] of DNA (since [[prokaryotic]] cells lack nuclei, mRNA produced is immediately translated into amino acid chain)
				- ![[Pasted image 20230822101521.png]]
			- in eukaryotes, takes place in nucleus; during process, mRNA synthesized along template DNA strand
				- ![[Pasted image 20230822101539.png]]
				- mRNA then moves from nucleus to cytoplasm via [[nuclear pores]]
		- [[translation]]: synthesis of [[polypeptides]] (singular) using genetic info in messenger RNA
			- in eukaryotic cells, takes place in cytoplasm, more specifically in the [[ribosome]]s
				- transfer RNA molecules = bring *specific* amino acids to growing polypeptide strand; sequence dictated by sequence of bases along mRNA strand
	- DNA = contains all info for protein synthesis + can be compared to the "master copy" of the info
		- for [[eukaryotic]] cells, DNA protected in nucleus
- ### 6.5.2: Transcription: RNA Formation from the DNA Template
	- RNA as nucleic acid polymer = made up of nucleotide subunits
		- has ribose as sugar, DNA had deoxyribose
		- ![[Pasted image 20230822101950.png]]
			- ribose: two hydroxyl groups
			- deoxyribose: one hydroxyl group
		- had nitrogenous base [[uracil]] while DNA has [[thymine]]
	- during [[transcription]], growing mRNA synthesized w/ help of enzyme ([[RNA polymerase]]) which links the ribonucleotides together
		- [[transcription initiation complex]] = consists of RNA polymerase and [[protein]] transcription factors bound to DNA double helix (which then unwinds, causing RNA synthesis to begin at the start point of template strand)
			- ![[Pasted image 20230822102631.png]]
		- [[messenger RNA]]: [[RNA polymerase]] directs synthesis of nucleotides that're complementary to DNA template; like DNA, enzyme adds nucleotides to 3' OH group of existing strand, so new messenger RNA strand grows in 5' to 3' direction
			- ![[Pasted image 20230822102608.png]]
		- begins when [[RNA polymerase]] recognizes [[promoter]] (defined below)
			- ![[Pasted image 20230822102659.png]]
			- **enzyme begins synthesizing mRNA in 5' to 3' direction; mRNA strand = [[antiparallel]] to DNA template**
		- both prokaryotic and eukaryotic DNA have a [[promoter]], or a specific nucleotide sequence binding RNA polymerase and showing where RNA transcription will begin
			- eukaryotes: TATA box (serves as recognition site for proteins that aid RNA polymerase binding) usually present in promoter, consisting of nucleotide sequences like TATAAA
				- collection of proteins ([[transcription factors]]) help RNA polymerase to recognize + bind to promoter
			- prokaryotes: RNA polymerase *specifically* recognizes and binds **directly** to the [[promoter]]
- ### 6.5.3: Transcription: Termination and RNA Protection
	- [[terminator sequence]]: sequence of nucleotides on the DNA strand signaling the end of [[transcription]]
		- ![[transcriptionend_termsequence.png]]
		- contains high concentration of [[adenine]] (a [[base]])
		- [[uracil]] in RNA pairs w/ adenine in RNA to form (relatively) weak [[hydrogen]] bonds between [[DNA]] and [[messenger RNA]] strands
			- bonds enable RNA strand to easily break away from template
	- after transcription, a series of [[enzymes]] in [[nucleus]] modify [[messenger RNA]] transcript before it undergoes translation in [[cytoplasm]]
		- during [[mRNA processing]], ends of transcript altered, sections of [[molecule]] removed, and remaining parts sewn back together
		- ![[mRNA_protection_and_processing.png]]
			- 5' end capped w/ methylated [[guanine]] [[nucleotide]] (cap protects [[messenger RNA]] from degradation by *hydrolytic* enzymes as it travels through cytoplasm before translation)
				- later, cap serves as signal for [[ribosome]] attachment
			- 3' end modified before transcript leaves nucleus
				- [[poly(A) tail]]: 30-200 [[adenine]] nucleotides (tail protects mRNA from degradation, like the cap of the 5' end, and assists it in export/leaving from the nucleus)
- ### 6.5.4: Posttranscriptional Modification/RNA Splicing
	- in [[eukaryote]]s: avg length of new [[messenger RNA]] transcript = abt 8000 nucleotides
		- only takes 1200 nucleotides to code for avg-sized protein of 400 amino acids
		- mRNA transcript + [[DNA]] template used to synthesize it = long, untranslated segments
			- most of these sequences occur between coding segments, so sequence codes for [[polypeptide]] = not continuous
	-  [[posttranscriptional modification]] = describes how newly synthesizes mRNA strands is processed
			- refers to addition of a 5' cap and a [[poly(A) tail]] to transcript
			- includes [[RNA splicing]]
			1. 5' cap and [[poly(A) tail]] added to newly transcribed mRNA molecule (cap and tail protect transcript from hydrolysis, signal ribosome attachment, and help export molecule from nucleus)
				- ![[posttrans_mods_step_1.png]]
			2. involves [[RNA splicing]]; mRNA transcript has long sequences that don't code for proteins called [[introns]]
				- ![[posttrans_mods_step_2.png]]
				- sequences that code for proteins = [[exons]]
				- introns found between exons
				- before transcript is translated into protein, [[introns]] removed (see illustration; short nucleotide sequences at ends of introns provide signal to identify splice sites)
					- splice sites then recognized by substances consisting of RNA and protein molecules called [[snRNPs]] (snurps)
						- ![[snRNPs_and_splice_sites.png]]
					- in mRNA splicing: RNA present in [[snRNPs]] functions as catalyst, role generally taken on by proteins of cell
						- idea of catalytic role for [[snRNPs]] arose from discovery of [[ribozymes]] (RNA molecules that function as enzymes)
	- on new mRNA strand, noncoding portions between coding portions = intervening sequences, otherwise known as [[introns]]
		- during mRNA splicing, [[introns]] removed and [[exons]] pasted together
		- splicing takes place before mRNA transcript leaves [[nucleus]]
		- several [[snRNPs]] join w/ additional [[proteins]] to form large complex called [[spliceosome]] (see illustration below)![[spliceosome_diagram.png]]
			- splices out intron and joins adjacent exons together
			- when introns removed + exons jointed together, modified mRNA molecules exported from nucleus into cytosol
## 6.6: Translation
- ### 6.6.1: Translation: Ribosomal and Transfer RNA
	- [[translation]]: synthesis of [[polypeptide]] strand using genetic info present in [[messenger RNA]] molecule; nucleotide language translated into [[amino]] acids
	- [[tRNA]] (transfer RNA) molecule functions as translator from nucleotide into amino acid language
		- ![[tRNA_diagram.png]]
		- img: 2D representation of tRNA
			- 5' and 3' ends of strand now adjacent due to folding
			- 3' end: site for amino acid attachment
			- middle loop contains [[anticodon]] sequences unique to each RNA transfer type
		- single, ~80 nucleotide-long RNA strand that folds back on itself to form  3D shape (which is maintained by [[hydrogen]] bonding between complementary [[nucleotide]]s in the strand)
		- specific [[anticodon]] on one end and an amino acid on the other end
		- [[anticodon]] sequence complementary to given sequence on mRNA strand (each sequence of mRNA translates into specific amino acid)
	- [[ribosome]]: cell organelle constructed in [[nucleolus]]
		- functions as site for [[protein]] [[synthesis]] in the [[cytoplasm]]
		- made up of 60% ribosomal [[RNA]] and 40% [[protein]]
		- exists as 2 subunits (see illustration)
			- ![[2_subunit_ribosome.png]]
- ### 6.6.2: The Role of Transfer RNA: Charging a tRNA Molecule
	- during [[translation]], [[tRNA]] [[anticodon]] binds to an [[mRNA codon]], specifying for a particular amino acid (important; helps provide correct amino acid for growing [[polypeptide]] strand)
		- structure of tRNA enables molecule to translate 2 languages of cellular activity: [[nucleotide]]s and [[proteins]]
			- structure enables tRNA to recognize correct [[mRNA codon]] specifying for a particular amino acid
		- ![[2D_tRNA_binding.png]]
		- illustration (a): 2D representation of tRNA
		- amino acid attaches at 3' end of molecule
		- anticodon = contains complementary sequences to 3 bases on mRNA strand
		- mistake could lead to defective protein (so problems in cell function)
	- process of *charging* a [[tRNA]] molecule involves bonding a tRNA molecule w/ a specific amino acid
		- ![[Pasted image 20230906151101.png]]
		- process by which [[aminoacyl-tRNA synthetase]] joins specific amino acid to a [[tRNA]] molecule
			- 1. active site of enzyme binds amino acid and an atp molecule
			- 2. atp loses 2 phosphate groups and attaches to amino acid as [[adenosine monophosphate]] (AMP)
			- 3. specific tRNA binds to amino acid + AMP drops off
			- enzyme releases aminoacyl tRNA complex or charged amino acid
		- occurs with help of enzyme: [[aminoacyl-tRNA synthetase]]
		- [[endergonic]] process
- ### 6.6.3: Translation: Initiation Events
	- 1961: [[Marshall Nirenberg]] cracked [[genetic code]], discovering that 3 mRNA nucleotides translated into one amino acid
		- experiments involves synthesis of artificial mRNA strands containing a known base sequence
		- e.g. strand of repeating uracil bases translated into a repeating strand of phenylalanine
		- conclusion: **the mRNA base pairs of UUU translated into phenylalanine***
	- [[codon]]: three nucleotide base-pairs of DNA or mRNA that codes for a specific amino acid or termination signal
		- - three-base code provides for 64 diff. amino acids (more than adequate!)
		- [[Marshall Nirenberg]]'s experiments verified: flow of info from gene to protein based on triplet code
		- ![[codon_img.png]]
		- unit of genetic code
			- more than 1 codon may specify for a particular amino acid
			- [[degeneracy]]: relative unimportance 3rd nucleotide plays in specifying for given amino acid (bc first 2 determine amino acid)
			- no [[ambiguity]], either (individual types of codons can only translate into one amino acid, but amino acids can form from different codons)
			- ![[codons_not_specific.png]]
		- [[mRNA codon]]s read from 5' end of strand to the 3' end of strand
	- 3 stages of [[translation]]:
		- [[initiation]]
			- ![[initiation_major_events.png]]
			- events include formation of a [[translation initiation complex]] (consists of large + small subunits of ribosome, mRNA, and tRNA)
			- energy expended in form of [[GTP]]
			- during initiation:
				- 5' cap on mRNA strand signals attachment of small ribosome unit
				- mRNA strand, tRNA molecule specifying for first amino acid, and both subunits of ribosome come together
				- small subunit bind sto both mRNA and tRNA
				- AUG sequence on mRNA signals translation
				- large subunits binds forming [[translation initiation complex]]
				- first tRNA (carries amino acid methionine) moves from A site to P site of large subunit
		- [[elongation]]
		- [[termination]]
- ### 6.6.4: Translation/Elongation: The Initiation of Elongation
	- during [[elongation]]:
		- amino acids added one by one to form polypeptide chain
		- elongation occurs through codon recognition, [[peptide]] bond formation, and [[translocation]]
			- addition of each amino acid involves aid of protein elongation factors
		- *major events*:
			- 1. codon recognition: img shows incoming aminoacyl tRNA bearing anticodon sequence "UUU"; binds to mRNA codon containing sequence "AAA"; occurs in A site of large ribosomal subunit
				- ![[translation_codon_recognition.png]]
			- 2. peptide-bond formation: img shows new amino acid being joined to existing polypeptide strand w/ help of RNA [[ribozymes]] (singular); [[peptide]] bond forms between the 2 amino acids; rRNA molecule of large ribosomal subunit (functioning as a [[ribozymes]]) catalyzes formation of peptide bond
				- ![[translation_peptidebond_formation.png]]
			- 3. translocation: after [[peptide]] bond forms, entire polypeptide strand moved to the [[tRNA]] in the A site; during this stage, tRNA in A site moves to the P site; tRNA in the P site moves into E site and is released from [[ribosome]]
				- ![[translation_translocation.png]]
- ### 6.6.5: Elongation Continued and Termination
	- during elongation stage of translation, new polypeptide synthesized by linking amino acids w/ [[peptide]] bonds
		- peptide bonds: link amino acids in polypeptide chain
			- formed by dehydration reactions and link the [[carboxyl]] group of one amino acid to the amino group of the next
			- water = byproduct of dehydration reaction
			- ![[peptide_bonds_elongation.png]]
	- [[termination]] of translation (elongation, specifically) occurs when a [[stop codon]] reaches A site of ribosome
	- ![[termination_of_elongation.png]]
		- [[protein release factor]] binds to [[stop codon]] and hydrolyzes bond between tRNA in the P site and the last amino acid of the polypeptide chain
			- 1. protein release factor's binding causes addition of a water molecule instead of an amino acid to polypeptide chain
			- 2. release factor hydrolyzes bond between tRNA in P site and last amino acid of polypeptide chain, and polypeptide is released from the ribosome
			- 3. complex of mRNA, tRNA, and ribosomal subunits dissociate + molecules can be recycled by cell to construct more proteins
		- polypeptide then released
## 6.7: Protein Synthesis Review
- ### 6.7.1: Polypeptide Destinations: Signal Peptides and ER Ribosomes
	- takes less than a minute to produce avg-sized polypeptide (hundreds of amino acids long!)
		- usually, a single [[messenger RNA]] strand used to make several copies of polypeptide at the same time (simultaneously), increasing the speed of polypeptide production
	- [[polyribosome]]: [[aggregation]] of several ribosomes attached to one mRNA molecule, increasing efficiency of polypeptide synthesis
		- ![[polyribosomes.png]]
		- (a): transmission electron micrograph of [[polyribosome]]s
		- (b): schematic representation of [[polyribosome]]s
			- during translation, first ribosome attaches to 5' initiation sequence of mRNA molecule
			- as ribosome moves in the 3' direction, a new incoming ribosome attaches to the 5' site
			- result: string of ribosomes in various stages of polypeptide synthesis
	- [[signal peptides]]: sequences of amino acids on polypeptides that target a protein to the ER ([[endoplasmic reticulum]])
		- polypeptides of proteins destined for [[endomembrane system]] marked by signal peptides
		- entire polypeptide-ribosome complex becomes embedded into ER membrane
		- once synthesis = done, polypeptide may be exported from cell (if it a [[secretory protein]]) or remain partially embedded in the ER membrane (if it is a [[membrane protein]])
	- ![[polypeptides_cytosol.png]]
		- synthesis of *all* proteins begins in cytosol where free ribosomes translate mRNA molecules
			- after translation, some proteins are bound to the [[cytosol]] whereas some will enter [[endomembrane system]]
	
		- ribosomes that carry ER-bound proteins bind to a receptor site of the ER and become [[ER ribosomes]]
		- polypeptide moves through ER membrane + enters ER compartment
- ### 6.7.2: Protein Synthesis: An Overview
	- [[translation]] involves the synthesis of a polypeptide as dictated by a mRNA nucleotide sequence
		- during translation, a messenger RNA strand, tRNA molecules, and ribosomal subunits come together
		- nucleotide codons on mRNA strand specify tRNA anticodons
		- tRNA molecules donate specific amino acids
		- ribosomal subunits provide physical site where translation occurs
		- **product**: newly synthesized polypeptide strand (which then undergoes protein synthesis)
	- translation takes place in 3 stages:
		- [[initiation]]
		- [[elongation]]
		- [[termination]]
	- during translation nucleotide codons on mRNA strand converted into amino acids
		- amino acids are bonded together to form a polypeptide strand
		- proteins = polypeptide strands in a 3D form
		- so, events of translation enable [[protein synthesis]]
	- steps:
		- 1. mRNA strand reading 5' AUGGAUCAACGUUAA 3' used to translate new polypeptide strand
			- ![[translation_step1.png]]
		- 2. specific type of tRNA molecule possesses anticodon 3' UAC 5' and the amino acid [[methionine]]; tRNA moves into the A site of ribosome and bonds w/ 5' AUG 3' on mRNA
			- ![[translation_step2.png]]
		- 3. w/ each new incoming tRNA, the complex of molecules shifts, and the result is that the ribosomal subunits move toward 3' end of mRNA strand
			- ![[translation_step3.png]]
		- 4. second tRNA molecule comes in and has the anticodon 3' CUA 5' and the amino acid [[aspartine]]; moves into the A site and binds w/ mRNA codon 5' GAU 3'; first tRNA molecule moves into P site; peptide bond forms between the two amino acids
			- ![[translation_step4.png]]
		- 5. third tRNA molecule attaches to A site bearing the anticodon 3' GUU 5' and the amino acid valine; it binds w/ the mRNA codon 5' CAA 3'; first and second tRNA molecules move into the E and A sites, respectively; growing polypeptide now three amino acids long
			- ![[translation_step5.png]]
		- 6. fourth tRNA molecule attaches to the A site carrying the anticodon 3' GCA 5' and the amino acid [[arginine]]; it binds w/ the mRNA codon 5' CGU 3'; the second and third tRNA molecules move into the E and A sites, respectively; chain is now four amino acids long
			- ![[translation_step6.png]]
		- 7. fifth tRNA molecule moves to the A site; has anticodon 3' AUU 5' and doesn't carry amino acid; looking at genetic dictionary, corresponding mRNA codon, 5' AUU 3' is a [[stop codon]]; tRNA exits via the E site
			- ![[translation_step7.png]]
## 6.8: The *lac* Operon
- ### 6.8.1: Control Mechanisms: Lactose Metabolism in *E. Coli*
	- both [[prokaryotic]] and [[eukaryotic]] cells have mechanisms to regulate DNA and RNA
		- process prevents cells from expending energy on unnecessary metabolic processes
		- one such mechanism found in [[E. coli]] (species of bacteria that inhabits our digestive tract)
			- bacteria employ a regulatory process to control the synthesis of three enzymes responsible for lactose metabolism
			- the [[lac operon]] is a regulatory unit on E. coli DNA that serves to control the synthesis of three enzymes responsible for metabolizing lactose
				- ![[gene_regulation.png]]
				- these enzymes:
					- ![[lactose_active_genes.png]]
					- [[B-galactosidase]]
						- breaks down lactose
					- [[permease]]
						- membrane protein that transports lactose into the cell
					- [[transacetylase]]
						- role unclear, but present nonetheless
				- in absence of lactose, these enzymes aren't produced
					- ![[lactose_genes_without_lactose.png]]
					- these enzymes (as proteins) are synthesized through the processes of transcription and translation + require an expenditure of cellular energy ([[endergonic]] reaction)
					- by regulating enzyme synthesis, cells conserve energy

- ### 6.8.2: Jacob and Monod's Model: The *lac* Operon
	- in prokaryotes, mRNA transcription requires a [[promoter]]
		- promoter serves as a recognition site for [[RNA polymerase]] (which binds to the promoter and proceeds to direct transcription of the [[messenger RNA]] molecule)
	- experiments conducted by [[Francois Jacob]] and [[Jacques Monod]] determined mechanisms underlying the [[lac operon]] (which regulates mRNA transcription of enzymes responsible for digesting lactose)
		- identified [[operator site]] (sequence of bases that's located between genes and promoter)
			- ![[prokaryotic_gene_regulation.png]]
			- *hypothesized* that when a specific protein occupies operator site, RNA polymerase unable to bind to the promoter (thus, transcription wouldn't occur)
			- since protein prevents transcription, it's called a [[repressor protein]]
				- in absence of lactose, repressor protein binds to operator and thus transcription of lactose-digesting enzymes is prevented
		- as lactose enters cell, it's converted into allolactose (prevents the repressor protein from binding to the operator)
			- RNA polymerase binds to the promoter and transcription of the three enzymes occurs
			- [[lac operon]] is an [[inducible operon]] (meaning that it's stimulated when a specific molecule interacts w/ a regulatory protein)
			- ![[lac_operon_inducible.png]]
		- basic components of [[lac operon]]: ![[lac_operon_basic_components.png]]
		- **single stretch of DNA regulates and synthesizes lactose-digesting enzymes**
		- 3 enzymes responsible for digesting lactose:
			- [[B-galactosidase]] (beta galactosidase)
			- [[permease]]
			- [[transacetylase]]
			- gene coding for them found adjacent to one another on DNA strand and are called lacZ, lacY, and lacA, respectively
	
- ### 6.8.3: lac Operon: The Summary
	- an [[operon]] is a unit of function consisting of regulated clusters of genes
		- provides control of a sequence of genes that're related in metabolic activity
		- [[lac operon]] = [[inducible operon]] that controls the transcription of 3 enzymes that metabolize lactose
		- ![[lac_operon_summary.png]]
			- wrap-up:
				- 1. looking at img, you can see that gene sequences coding for the 3 enzymes occur adjacent to each other on the DNA strand. the sequences are labeled lacZ, lacY, and lacA on the illustration
				- 2. [[promoter]] sequence = to the left of the sequences, enabling RNA polymerase to synthesizes mRNA in a 5' to 3' direction
				- 3. genetic sequence to the left of the promoter, called the lacI gene, codes for the [[repressor protein]]
				- 4. when lactose is absent, repressor protein binds to the [[operator site]], which lies in between the promoter and the gene sequences of the enzymes
				- 5. location of the operator enables [[repressor protein]] to act as a physical block, preventing RNA polymerase from binding to the promoter
				- 6. when lactose is present, the repressor protein = inhibited and [[transcription]] of the enzyme lactase begins
## 6.9: Eukaryotic Genomic Organization
- ### 6.9.1: The Eukaryotic Genome: DNA Packing
	- both prokaryotic and eukaryotic cells must organize + control genetic material
		- prokaryotic genomes = millions of nucleotides
		- eukaryotic [[genome]] = in orders of magnitude *larger*
	- [[chromosomes]]: highly compacted forms of DNA and associated proteins
		- human cell contains 46 chromosomes
		- each chromosome has approximately 2 * 108 nucleotides!!!
		- if chromosomes in one [[diploid]] human cell were stretched out, the strand would be abt 2.8 meters long!!!
			- much of this DNA isn't active and doesn't code for proteins, but must still be copied during cell division
		- eukaryotic cells package their chromosomal DNA into [[chromatin]]
			- during [[interphase]], the chromatin exists as a diffuse mass
			- during [[mitosis]] and/or [[meiosis]] of the cell cycle, a cel's chromatin is condensed to form recognizable bodies of chromosomes that can be viewed under a light microscope
		- levels of DNA packing:
			- 1. [[nucleosome]]: consists of DNA wrapped around a core of proteins called [[histone]]s; each protein core is composed of eight histones, two of each: H2A, H2B, H3, and H4
				- ![[nucleosome_structure.png]]
			- 2. string of nucleosomes coils together w/ help of histone H1 to form a 30-nm [[chromatin]] fiber
				- ![[30nm_chromatin_fiber.png]]
			- 3. 30nm [[chromatin]] fiber folded and anchored to scaffold of nonhistone proteins, resulting in a configuration known as a [[looped domain]]
				- ![[looped_domain.png]]
			- 4. "beads on a string" configuration of nucleosomes = folded upon itself to form the 30nm [[chromatin]] fiber, which then forms looped domains anchored on a protein scaffold
				- looped domains further folded together to form [[chromosomes]] that are visible under a light microscope during metaphase of cell division
				- ![[chromosome_formation_from_chromatin.png]]
- ### 6.9.2: Eukaryotic Genomic Organization: Repetitive DNA
	- in prokaryotes, most of the DNA in a genome codes for protein products
		- small amt exists as noncoding regulatory sequences like [[promoter]]s
		- in contrast, 90% of sequences in eukaryotic genomes *don't* sequence for protein products
			- some of these noncoding sequences are made up of [[repetitive DNA]]
				- 2 major types: *tandemly* repetitive DNA and *interspersed* repetitive DNA
					- *tandemly*: accounts for 10-15% of mammalian DNA
						- lengths of units are only 1-10 base pairs long, but sequences may be repeated hundreds of thousands of times, one after the other
						- *often found within [[centromeres]] and [[telomeres]]*
						- also referred to as *satellite DNA* bc when genomic DNA is cut up into segments and centrifuged, pieces containing tandemly repetitive sequences will form a *band* separate from the rest of the DNA segments
							- ![[tandemly_repetitive_DNA.png]]
					- interspersed: accounts for 25-40% of mammalian DNA
						- length varies from 100-10,000 base pairs
						- scattered throughout genome unlike tandemly repetitive DNA, which occurs one after the other
							- often, dispersed sequences similar rather than identical to each other
							- ![[interspersed_repetitive_DNA.png]]
	- scientists can determine whether or not a sequence of DNA has multiple copies by denaturing the DNA + observing how fast the strands reanneal
		- DNA sequences that contain multiple repeats reanneal faster than sequences that exist as a single copy

- ### 6.9.3: Eukaryotic Genomic Organization: Gene Families
	- gene families: repeating DNA sequences that code for identical or very similar RNA products, located close together on a chromosome
		- 3 examples:
			- *Alu* gene family
				- multiple copies of gene, each 300 base pairs long, transcribe for an RNA molecule
			- ribosomal RNA (rRNA) gene families
				- ![[rRNA gene families.png]]
					- 1. strand at top of illustration = DNA template that codes for rRNA gene family
					- 2. strand of mRNA transcribed (2nd strand in img); RNA transcript contains related genes coding for rRNA; copies separated by noncoding spacer sequences
					- 3. RNA processing yields 3 rRNA molecules: 18S, 5.8S, and 28S (S strands for respective sedimentation rates in an [[ultracentrifuge]]; larger molecules sediment at slower speeds)
				- present in multiple copies since cell has high demand for ribosomal RNA
			- globin gene families

- ### 6.9.4: Eukaryotic Genomic Organization: Transposons and Amplified Genes
	- gene families thought to have arisen from duplication of a parent gene
	- [[pseudogenes]]: copies of existing genes that lack regulatory sequences like promoters + often found in [[gene families]]
	- [[gene amplification]] takes place when many copies of a gene = synthesized in order to increase production of the gene product
		- examples:
			- salamander eggs
				- before fertilization, millions of copies of rRNA genes are synthesized
				- amplified genes exist as tiny DNA circles separate from the chromosome
				- after fertilization, copies become active and act as template for rRNA transcription
				- newly developing embryo needs multitude of rRNA in order to synthesize vast amt of protein products it'll require
			- cancer cells
				- [[gene amplification]] may occur in cancer cells of patients who have undergone chemotherapy; drugs kill many cells, but some are resistant
					- cells have been found to contain amplified sequences of the genes responsible for drug resistance
	- [[transposons]]: segments of DNA that're capable of moving within a genome
		- make up over 50% of corn genome
			- ![[transposons diagram.png]]
		- general transposon contains DNA sequence coding for enzyme [[transposase]] that's flanked by [[inverted repeats]]
			- [[transposase]] functions by making staggered cuts within a strand of DNA + allowing a copy of the entire transposon sequence to be inserted into the cut site
	- [[retrotransposons]]: product [[reverse transcriptase]], which can make a new copy of DNA from RNA and insert it into the genome
		- ![[retrotransposons.png]]
		- transposons that move by means of an RNA intermediate that's formed from a transcript of the retrotransposon DNA
		- to be inserted into genome, RNA intermediate must be converted back to DNA
			- accomplished by enzyme [[reverse transcriptase]] (which is coded for in the retrotransposon)
			- another enzyme that's coded for in the retrotransposon will insert resulting DNA into new site

## 6.10: Controlling Protein Synthesis in Eukaryotes
- ### 6.10.1: Eukaryotic Gene Control: Transcriptional Controls
	- gene = distinct unit of hereditary info composed of a specific nucleotide sequence in DNA
		- all of the cells of an organism contain the same genes, but don't necessarily *use* the same genes
		- eukaryotic cells employ a number of control mechanisms to ensure that genes are expressed *only* when they're needed
		- several control points during gene expression:
			- pretranscriptional (involving physical availability of genes):
				- [[DNA methylation]]: addition of methyl groups (-CH3) to an intact DNA strand, typically to the [[cytosine]] bases; results in inactivation of DNA
				- [[histone acetylation]]: attachment of acetyl groups (-COCH3) to specific amino acids of histones
					- acetylation changes histone's conformation and loosens its grip on DNA
					- acetylated DNA is more readily transcribed
				- transcriptional controls involve [[transcription factors]], which are necessary in order for transcription to occur
					- ![[transcription factor review.png]]
					- review: promoter = sequence of DNA near gene that regulates eukaryotic gene transcription
						- RNA polymerase requires presence of transcription factors in order to bind to the promoter; one transcription factor recognizes the TATA box within the promoter while the other transcription factors recognize proteins (including each other and RNA polymerase)
						- transcription only proceeds when [[transcription initiation complex]] is formed
					- have DNA-binding domain, which is part of its 3D structure and allows it to bind to DNA
						- 3 main types of DNA-binding domains:
							- ![[3 types of DNA-binding domains.png]]
							- helix-turn-helix (top left)
								- notice how the 2 cylindrically shaped portions of a transcription factor, which are alpha helices, wrap around the DNA strand
							- zinc finger
								- consists of repeating units of an alpha helix + beta pleated sheet held together by a zinc atom
								- these repeating units wrap around DNA strand
							- leucine zipper
								- consists of 2 polypeptide chains that consist of many leucines
								- chains coiled around each other, bringing 2 polypeptides together
		- [[enhancers]]: sequences of [[DNA]] that are up to thousands of nucleotides away from the [[promoter]]
			- ![[enhancers.png]]
			- unlike promoters, enhancer physically farther away from the gene that it controls + can be either upstream (toward the 5' end) or downstream (toward the 3' end) from the gene
			- transcription factors called [[activators]] bind to enhancers + help form the [[transcription initiation complex]]
				- TIC forms when DNA folds over and activators bind to transcription factors at promoter
- ### 6.10.2: Eukaryotic Control Mechanisms: Posttranscriptional and Posttranslational Controls
	- review:
		- gene expression can be controlled at the level of pretranscription or transcription
		- examples of pretranscription control includes DNA methylation and histone acetylation
		- transcription-level controls include transcription factors and enhancers
	- gene expression can also be controlled at the level of:
		- ![[pre, mid, and post-translation control.png]]
		- post-transcription:
			- after transcription yields [[messenger RNA]] molecule, cell still has many other opportunities to regulate protein processing
			- review: RNA processing involves addition of a cap and a tail to a new mRNA transcript
				- this addition prevents nuclease enzymes present in the cytoplasm from hydrolyzing the transcript
			- one hypothesis: build-up of protein product can wear down protective additions, enabling the break down of the mRNA transcript and preventing translation
		- pre-translation:
			- review: during initiation of translation, certain proteins (not pictured in illustration) must be physically present for translation to occur
			- deactivating initiation factors will prevent translation
		- translation:
			- translation of mRNA strands may be blocked by attachment of regulatory proteins that bind to sequences near the 5' end of the mRNA strand and prevent the attachment of ribosomes
		- post-translation:
			- [[proteasomes]]: giant complexes of proteins functioning to break down the unneeded proteins of a cell
				- ![[proteasomes + post-translate control.png]]
				- img depicts break down of proteins by a proteosome enzyme. steps are as follows:
					- enzymes in cytosol attach [[ubiquitin]] molecules to the protein to be degraded, thereby marking it
					- a proteosome recognizes the ubiquitin and degrades the protein
- ### 6.10.3: Prokaryotes vs. Eukaryotes: Protein-making Machinery
	- prokaryotes + eukaryotes have many physiological differences that lead to differences in protein synthesis, including:
		- presence or absence of a nucleus
			- eukaryotic cells = double-membrane nucleus that encloses chromosomes, and they have membrane-bound organelles
			- prokaryotes don't have a nucleus or membrane-bound organelles; instead, they have a region called the [[nucleoid]], where the chromosome is found
		- chromosomal differences
			- DNA of eukaryotic cells = organized into rod-like chromosomes
			- DNA of a prokaryotic cell = organized into a singular circular chromosome
				- in general, prokaryotes have less DNA than eukaryotes
		- promoter recognition
			- RNA polymerase depends on transcription factors to recognize the promoter (eukaryotic cell)
			- prokaryotic RNA polymerase recognizes promoter w/o aid of transcription factors
		- method of transcription termination
			- eukaryotes: transcription proceeds through stop sequence and ends abt 10-35 nucleotides later
				- as seen in img above, mRNA that results from eukaryotic transcription extends past termination signal, an AU-rich region
			- prokaryotes: transcription stops at the end of the stop sequence
		- ribosome size
		- location of transcription and translation
			- ![[pro vs euk - transcription and translation location.png]]
			- eukaryotic cells: both transcription and RNA processing happen in nucleus; translation happens in cytoplasm
			- prokaryotic: no nucleus; transcription and translation performed in cytosol of cell; also don't undergo any RNA processing
		- protein targeting
			- eukaryotic proteins: signal peptide targets protein to the ER; receptor on the ER recognizes the signal
				- ![[eukaryotic protein signaling.png]]
				- diagram above shows insertion of a protein (from right to left) into the ER of a eukaryotic cell
			- bc prokaryotes have no membrane-bound organelles, they have no complicated system for targeting proteins into organelles