## 6.1: Discovering DNA
- ### 6.1.1: Molecular Genetics: the Protein vs. DNA Debate
	- [[Gregor Mendel]]: described factors that were responsible for passing traits on; didn't know what the factors were
	- coordinated scientific effort to identity genetic material = mid-1800s to mid-1900s
		- at beginning, researchers formalized qualities that the unknown genetic material must possess; 4 qualities:
			- 1. information
				- must store relatively large amt of info, enough to produce cell's [[proteins]] and other structures (bc DNA makes RNA makes proteins)
			- 2. replication
			- 3. communication
			- 4. change
		- **these 4 qualities make up the "brain" of the cell**
	- during 1800s and early 1900s, [[Friedrich Miescher]], [[Robert Feulgen]], and [[Archibald Edward Garrod]] contributed to understanding of the genetic material
		- [[Friedrich Miescher]]: extracted an acidic material from white blood cells' nuclei, calling it [[nucleic acid]]
			- #### type of [[nucleic acid]] found in chromosomes of all [[eukaryotic]] cells: [[DNA]]
			- later, was determined that [[nucleic acid]]s are made up of 4 [[nucleotide]]s: adenine, guanine, cytosine, and thymine
			- ![[Pasted image 20230816080801.png]]
		- [[Robert Feulgen]]: developed staining technique that could *quantify* DNA; the more DNA present in the nuclei, the darker the stain
		- ![[Pasted image 20230816081940.png]]
			- w/ this, researchers found that **DNA levels are species-specific**
			- across all species, sex cells consistently contained *half* the amount of DNA that the [[autosomal]]/unrelated to sex cells did
		- during this time, most scientists thought that [[protein]] (not [[DNA]]) was genetic material (even w/ evidence)

- ### 6.1.2: Continuing to Link Genes to Chemicals: Muller, Beadle, and Tatum
	- in 1920s: Hermann Muller and Thomas Hunt Morgan furthered genetics field
		- Muller: could increase mutation rate in fruit flies by subjecting them to x-rays
		- ![[Pasted image 20230816082422.png]]
			- 10C increase in temp = doubled mutation rate
	- George Beadle and Edward Tatum: subject red bread mold to x-rays, **greatly increasing mutation rate**
	- ![[Pasted image 20230816082431.png]]
		- normally mold can synthesize almost all of the organic compounds it needs + can be grown on [[minimal medium]] containing a few minerals, 1 vitamin, and an energy src
		- work:
			- 1. spores subjected to x-rays; placed on enriched media containing amino acids and allowed to grow until they produce spores that're collected
			- 2. spores crown on [[minimal media]]; if they can't, then a mutation has occurred bc of x-rays
			- 3. pieces of mutant mold = put in tubes containing minimal media +1 amino acid each; mutant strain can only grow when amino acid [[alanine]] present in tube
			- result indicated that mutation could be linked to defect in specific enzyme (genes + enzymes linked); "one gene one enzyme"

- ### 6.1.3: Griffith and Transformation
	- 1928: [[Frederick Griffith]] performed series of experiments w/ 2 strains of *Pneumococcus* bacteria
	- ![[Pasted image 20230816083107.png]]
		- smooth (S) strain: known to be [[virulent]] (deadly)
			- smooth colony type; mice die when injected
			- when heated/killed + injected, mice live (causes chemical change)
			- ![[Pasted image 20230816083059.png]]
		- rough (R) strain: [[avirulent]] (not deadly)
			- produced colonies w/ rough surface + type; mice live when injected
		- when heat-killed smooth *and* normal rough strai
		- ![[Pasted image 20230816083042.png]]ns are both injected, mice die
			- **conclusion: substance present in heat-killed smooth transformed living (rough) strain into smooth bacteria**
				- transforming substance: then thought to be genetic material, but not yet identified

- ### 6.1.4: Avery, MacLeod, and McCarty/Hershey and Chase: DNA Wins!
	- 1944: [[oswald avery]], [[colin macleod]], and [[maclyn mccarty]] showed that *nucleic acids isolated from heat-killed deadly (S) strain of a [[bacteriophage]]* responsible for *transforming harmless (R) strain deadly*
		- thus, **DNA, not protein, is genetic material**
	- 1952: [[alfred hershey]] and [[martha chase]]: series of experiments involving reproduction of [[bacteriophage]]s, (viruses that infect bacteria)
	- ![[Pasted image 20230816083554.png]]
		- hershey and chase capitalized on fact that *viruses are simple structures w/ 2 basic molecules: a protein coat and DNA*
			- one **had** to be genetic material, capable of using bacteria to produce more viruses
			- sulfur in [[bacteriophage]]: 35S; phosphorus in [[bacteriophage]]: 32P
		- during viral reproduction, protein coats *remained on the outside of the bacterial cells + could be separated w/o affecting viral replication*
			- new viruses produced in bacterial cells = radioactive DNA
		- **demonstrated that DNA is the genetic material**!!!

- ### 6.1.5: Chargaff and Franklin and Wilkins: The DNA Story Begins
	- DNA established as genetic material, so scientists insensified efforts to *characterize DNA's molecular structure*
		- structure must allow 4 basic functions of genetic material (see 6.1.1) which include storage of **large quantities of information**
			- achieved by 4 nucleotide subunits (illustrated below)
			- ![[Pasted image 20230816084015.png]]
				- 2 subuntis: [[purine bases]] (adenine and guanine)
				- other 2 subunits: [[pyrimidine bases]] (thymine and cytosine)
	- 1950: Erwin Chargaff found that **total amount of DNA: SPECIES-SPECIFIC**
		- within a given species amt of adenine relative to thymine = equal, as was amt of cytosine relative to guanine
	- Rosalind Franklin + Maurice Wilkins: used x-ray diffraction to study DNA structure
	- ![[Pasted image 20230816084340.png]]
		- technique involves striking DNA crystals w/ x-rays and photographing resulting diffraction pattern
		- **crisscross pattern provides evidence that DNA = helical in shape**
			- other scientists then used these measurements to furhter their understanding of DNA structure + function

## 6.2: DNA Structure Revealed
- ### 6.2.1: Watson and Crick: The Clues
	- [[James Watson]] and [[Francis Crick]] = first to *accurately* describe [[DNA]] structure
		- constructed hypothetical model that incorporated the Erwin Chargaff's findings + findings of [[Rosalind Franklin]] and [[Maurice Wilkins]]
			- 3 sets of data:
				- 1. chargaff's rules: erwin chargaff found that the amt of adenine and thymine are equal (and the amount of guanine and cytosine) are equal within a species
				- 2. repetitive amt ratios: franklin & wilkins' data showed that some element repeated every 0.34 nm and another repeated every 3.4 nm
				- ![[Pasted image 20230816123533.png]]
					- molecule ~2nm wide
					- prob ties into chargaff's rules
				- 3. shape of molecule: based on [[diffraction]] pattern, DNA molecule appeared helical in shape
			- based on 3 sets of data, proposed that **DNA was a double-stranded helix, each strand consisting of a series of [[nucleotide]]s**
				- data showed that the 2 strands = *exactly* 2 nm apart (a)
				- ![[Pasted image 20230816123911.png]]
				- (b) shows that each strand consists of nucleotide subunits of *a sugar, a phosphate, and a base*; these units are 0.34 nm apart
				- ![[Pasted image 20230816123927.png]]
					- sugar + phosphate form backbone of strand, and base available for hydrogen bonding with base on other strand
				- to remain consistent w/ width data, necessary for a [[pyrimidine]] (1-ring structure) to **always** pair w/ a [[purine]] (2-ring structure)
					- adenine (A) must bond *only* with thymine (T) and guanine (G) only with cytosine (C)
						- specific bonding between adenine and thymine + between guanine and cytosine *fits Chargaff's results!*
		- [[Linus Pauling]]: contemporary of watson + crick; proposed that DNA consisted of 3 nucleotide strands (incorrect; watson and crick's version proved to be correct)

- ### 6.2.2: Watson and Crick: The Double Helix
	- 4/25/1953: Watson and Crick's article "The Molecular Structure of Nucleic Acids: A Structure for DNA" appeared in journal *Nature*
		- understanding DNA structure enabled scientists to start untanglingg *process of how DNA serves as genetic material for cells*
	- DNA = described as a double helix bc composed of 2 [[nucleotide]] strands wound around each other
	- ![[Pasted image 20230817024531.png]]
		- (a): single nucleotide subunit
			- phosphate group, deoxyribose sugar, and 1 of 4 nitrogenous bases
			- carbons of sugar = numbers 1-5
			- each DNA strand made up of [[polynucleotide]] chain
		- (b): DNA molecule: double helix of polynucleotide chains
			- 2 strands = 2 nm apart
			- within strand, nucleotides .34 nm apart
			- 10 nucleotides in each turn of helix, so each turn occurs at 3.4 nm
			- repetition!
		- overall shape of molecular = comparable to twisted ladder
## 6.3: Introduction to DNA Replication
- ### 6.3.1: Replication: Meselson and Stahl
- 1958: Mathew Meselson and Franklin Stahl = experiments that tested the way  DNA replicates itself
	- bacteria = grown in heavy-nitrogen (15MN) medium for many generations
		cells then transferred to light-nitrogen (14N) medium and allowed to replicate for either one or two generations in light medium; process of *density gradient centrifugation* used 		to analyze weights of all 3 types of cells
	- Watson + Crick's model of double-stranded DNA molecule suggested that ***pair of nucleotide strands would separate and each would serve as template for new strand***
		- mechanism: [[semiconservative replication]]
	- competing w/ semiconservative model were *conservative* and *dispersive replication*
		- hypotheses of all 3: predicted diff weights for first/2nd gen cells
			cells subjected 2 one round of replication = bands intermediate in density, so DNA molecules consisted of 1 parental (heavy) and one new (light) strand
			therefore, conservative replication ruled out. cells subjected to 2 rounds = 1 intermediate and one light band, so dispersive replication ruled out
			density of cells in all 3 gens match pattern expected for *semi-conservative replication*
		- conservative: 2 parent strands wouldn't separate, replucation results in DNA molecular w/ 2 new strands
		- dispersive: parts of both parental strands separate + act as templates 4 new strand synthesis
- ### 6.3.2: DNA: Polymerization with Triphosphate Nucleotides
	- when a cell *divides* via [[semiconservative replication]], each new cell gets one original [[parental strand]] that serves as a template for the synthesis of a new [[complementary strand]]
		- this synthesis = DNA [[polymerization]]
			- [[polymerization]]: [[endergonic]] process, requiring expenditure of [[ATP]]
	- [[nucleotide triphosphate]]: contains energy in bonds that link 2nd and 3rd phosphate groups (which is then used to join the nucleotide to the [[growing strand]])
		- as part of new strand, [[nucleotide]] only has one phosphate group; nucleotides joined together by 3', 5' [[phosphodiester]] linkages
			- 3' and 5' designations refer to 3' and 5' carbons of the sugar molecule
			- ![[Pasted image 20230820131942.png]]
		- overall direction ([[orientation]]) of new synthesized strand = denoted by *its top and bottom nucleotides*, respectively
			- outer ends: not involved in phospho-diester linkage, resulting in empty 5' and 3' carbon
			- orientation of new strand said to be 5'-3' (orientation of [[complementary strand]] opposite of the [[parental strand]], meaning that they're [[antiparallel]])
			- ![[Pasted image 20230820132141.png]]


## 6.4: Events of DNA Replication
- ### 6.4.1: Events at the Replication Fork: The Leading Strand
	- [[prokaryotic]] cell = polymerizes 500 [[nucleotide]]s/sec
		- chromosome: circular, so replication occurs **efficiently**
	- [[eukaryotic]] cell = *synthesizes* 50 [[nucleotide]]s/sec
		- contains between 4-5 billion nucleotides(!)
		- use multiple [[replication bubble]]s, which greatly speed up replication
			- each [[replication bubble]] = two y-shaped [[replication fork]]s
	- DNA replication = *continuous* in one strand and *discontinuous* in the other
		- events basic to DNA process replication + specific to [[leading strand]]:
			- replication occurs at sites on molecule called [[replication bubbles]]; both strands replicated simultaneously at [[replication fork]]
				- ![[Pasted image 20230820132604.png]]
			- **overall events by which both strands are replicated**:
				- 1. strands separated and unwound by DNA [[helicase]]
				- 2. strands prevented from coming together by [[helix-destabilizing proteins]]
				- 3. enzymes that catalyze [[synthesis]] of new [[complementary strand]]s = [[DNA polymerases]]
					- can only add nucldeotides to the 3' end of [[growing strand]]
				- 4. [[RNA primer]] initiates synthesis of either new strand to provide first 3' end that enzyme requires
					- RNA primer = layer removed and substituted w/ DNA
				- synthesis of both complementary strands occurs in 5' to 3' direction
				- ![[Pasted image 20230820132955.png]]
					- strand growing towards replication fork grows continuously + called [[leading strand]] (which is [[antiparallel]] to the [[parental strand]] that served as its template)
- ### 6.4.2: Events at the Leading Strand, Part II
	- [[replication bubble]]: occurs at *origin of replication* and consists of 2 [[replication fork]]s
		- [[replication]] happens in opposite directions along each of replication forks (bubble enables DNA replication to occur in 2 directions)
			- [[DNA polymerases]] attaches to each replication fork
				- can **only** build new strands in the 5' to 3' direction (see img) so new strands synthesized in opposite directions
				- requires open 3' OH group to attach a new nucleotide; when just starting, that OH group is provided by 3' end of [[RNA primer]] strand that had been added by [[primase]]
					- first DNA nucleotide = added to open 3' OH group
					- nucleotides coming after = added to first DNA nucleotide
					- illustration shows new nucleotide that'll join strand; orientation of new strand opp. of orientation of [[parental strand]], so strands are [[antiparallel]]
						- ![[Pasted image 20230821121814.png]]
		- DNA strands = antiparallel (5' to 3' orientation runs in opp. directions)
			- [[leading strand]]: grows/synthesized continuously toward replication fork
			- [[lagging strand]]: grows/synthesized opp. of direction of replication fork
	- at each replication fork, og DNA strands = unwound (seen in img) in overall direction of unwinding and replication
		- img demonstrates [[antiparallel]] nature of [[DNA]] strands
		- ![[Pasted image 20230821115643.png]]
- ### 6.4.3: Events at the Replication Fork: The Lagging Strand
	- [[DNA polymerases]] add [[nucleotide]]s to the 3' end of a [[growing strand]], so only one of the new [[complementary strand]]s can grow continuously in the direction of the moving [[replication fork]]
		- [[leading strand]]: grows continuously
		- [[lagging strand]]: grows away from replication fork
	- the [[lagging strand]]:
		- discovery attributed to [[Reiji Okazaki]]
		- as the [[replication fork]] opens, [[leading strand]] comprises a continuous strand of new nucleotides growing in opening fork's direction
		- grows away from opening fork
			- ![[Pasted image 20230821123233.png]]
		- the two strands grow in opp. direction bc [[DNA polymerases]] can *only* add nucleotides to the 3' end of a strand
			- ![[Pasted image 20230821142926.png]]
			- as fork unwinds, [[lagging strand]] synthesized in 100-1000 nucleotide pieces called [[Okazaki fragment]]s
				- each [[Okazaki fragment]] = needs [[RNA primer]] to initiate synthesis
				- primers removed, [[DNA polymerases]] fills in gap w/ DNA nucleotides
				- fragments then joined together by [[ligase]]
		- *each of the [[parental strand]]s* serves as **template** for synthesis of a new [[complementary strand]]
		- #### end product of [[DNA replication]] is two *identical* DNA molecules
		- ![[Pasted image 20230821143108.png]]
			- question: is the final product 'flipped' bc of [[antiparallel]] strands?
- ### 6.4.4: Proofreading, End Replication, and [[Telomeres]]
	- DNA polymerase = synthesizes new nucleotides w/ great accuracy (ex. human genome is very, very complex, but roughly only *one nucleotide in a billion* is paired incorrectly)
	- [[DNA polymerases]]: responsible for [[genetic proofreading]]
		- works in conjunction w/ [[ligase]] to excise + replace incorrectly matched base pairs
	- DNA in a [[eukaryotic]] cell = rod-shaped, presenting a *problem* in DNA replication
		- [[telomeres]] come into play here
	- when growing [[lagging strand]] reaches end of molecule, last few DNA molecules can't be added
		- after RNA primer = removed, [[DNA polymerases]] can't fill gap w/ DNA bc DNA polymerase has no 3' OH end to attach to
		- DNA molecules shorten after each round of replication
			- ![[Pasted image 20230821143722.png]]
		- to avoid loss of genetic material, ends of DNA consist of long, repeating nucleotide strands called [[telomeres]] that don't code for cell products
		- [[telomeres]]: 
			- protect coding sequences of DNA
			- after many rounds of replication + shortening of the ends, noncoding sequences disappear
			- important coding sequences will be degraded
			- **certain cell lines possess enzyme called** ***telomerase*** **that can polymerize telomere sequences**
			- ![[Pasted image 20230822082414.png]]
				- cells that replicate many times, like [[germ cells]], have [[telomerase]]
			- typical repeated unit in human DNA = TTAGGG
			- ![[Pasted image 20230821144029.png]]

- ### 6.4.5: DNA Replication: A Summary
	- [[DNA]] consists of double helix of nucleotide strands (nucleotide subunit composed of sugar, phosphate, and either a purine or pyrimidine base)
		- within a strand, [[phosphodiester]] bonds join the nucleotides
			- sugars and phosphates form strand backbone
			- ![[Pasted image 20230822091134.png]]
		- between strands, specific [[base pairing]] occurs
		- ![[Pasted image 20230822091114.png]]
			- adenine and guanine = [[pyrimidine]]s (purines bond w/ pyrimidines)
			- adenine only bonds with thymine, and guanine bonds only with cytosine through hydrogen bonding
			- specificity = strands are *complementary* (see [[complementary strand]])
			- adenine, guanine, cytosine, and thymine are the nucleotides purine or pyrimidine base
	- initiation of replication happens when [[hydrogen]] bonds between strands break
	- ![[Pasted image 20230822090946.png]]
		- new strands synthesized in opp. directions of each other + called [[leading strand]] and [[lagging strand]]
		- molecule replicates via [[semiconservative replication]], w/ each strand acting as a template for the synthesis of a new [[complementary strand]]

## 6.5: Transcription
- ### 6.5.1: Transcription and Translation: An Overview
- ### 6.5.2: Transcription: RNA Formation from the DNA Template
- ### 6.5.3: Transcription: Termination and RNA Protection
- ### 6.5.4: Posttranscriptional Modification/RNA Splicing

## 6.6: Translation
- ### 6.6.1: Translation: Ribosomal and Transfer RNA
- ### 6.6.2: The Role of Transfer RNA: Charging a tRNA Molecule
- ### 6.6.3: Translation: Initiation Events
- ### 6.6.4: Translation/Elongation: The Initiation of Elongation
- ### 6.6.5: Elongation Continued and Termination

## 6.7: Protein Synthesis Review
- ### 6.7.1: Polypeptide Destinations: Signal Peptides and ER Ribosomes
- ### 6.7.2: Protein Synthesis: An Overview

## 6.8: The *lac* Operon
- ### 6.8.1: Control Mechanisms: Lactose Metabolism in *E. Coli*
- ### 6.8.2: Jacob and Monod's Model: The *lac* Operon
- ### 6.8.3: lac Operon: The Summary

## 6.9: Eukaryotic Genomic Organization
- ### 6.9.1: The Eukaryotic Genome: DNA Packing
- ### 6.9.2: Eukaryotic Genomic Organization: Repetitive DNA
- ### 6.9.3: Eukaryotic Genomic Organization: Gene Families
- ### 6.9.4: Eukaryotic Genomic Organization: Transposons and Amplified Genes

## 6.10: Controlling Protein Synthesis in Eukaryotes
- ### 6.10.1: Eukaryotic Gene Control: Transcriptional Controls
- ### 6.10.2: Eukaryotic Control Mechanisms: Posttranscriptional and Posttranslational Controls
- ### 6.10.3: Prokaryotes vs. Eukaryotes: Protein-making Machinery