
## 7.1: Plasmids and Gene Cloning
- ### 7.1.1: Biotechnology: Plasmids in Prokaryotes
	- [[biotechnology]]: the utilization of organisms and their components to perform functional tasks in research + to make products that are useful to humans
	- [[plasmids]]: small, circular pieces of DNA that're separate from bacterial [[chromosome]] and can carry [[genes]] that encode proteins w/ specific functions; self replicating
		- ![[plasmid_diagram.png]]
		- can code for:
			- the ability to form [[pili]], which are involved in [[bacterial conjugation]]
				- conjugation: process by which genetic material is transferred between two bacteria
				- plasmid can encode for a [[bacterium]]'s ability to form a [[pilus]] (plural: pili), which is a structure that connects two bacteria together and allows for the transfer of DNA from one bacterium to the other
				- ![[bacterial-conjugation-simple.png]]
			- antibiotic resistance: fungi produce chemicals called [[antibiotics]] that're able to kill bacteria (a natural competitor). bacteria that have a resistance plasmid are able to overcome specific antibiotics
			- **toxins**: some of the chemicals that bacteria produce can cause a person who has a bacterial infection to be ill; many of these chemicals are encoded in plasmids
			- **protective proteins**: some bacteria have protective protein coats that are encoded in plasmids
- ### 7.1.2: Using a Restriction Enzyme to Create a Vector
	- researchers can create specialized [[plasmids]] with a gene of interest inserted into them; these plasmids may then be placed into bacterial cells where they will be expressed
	- [[restriction enzymes]] or restriction [[endonucleases]] are enzymes that bacteria produce in order to cut foreign DNA  as a defense mechanism
		- ![[bacteriophage-inserting-DNA.png]]
			- [[bacteriophage]] (bacterial virus) inserting its DNA into a bacterial chromosome
			- to defend itself, bacterium uses [[restriction enzymes]] to cut the foreign DNA
			- once cut, DNA will be further broken down and used as building blocks for the bacterium's own use
		- cut DNA only at a specific sequence and most restriction enzymes make staggered cuts
		- resulting fragments have [[sticky ends]], which will be able to come back together due to of complementary base pairing between strands
		- biotechnologists often use the restriction enzyme [[EcoR1]], which makes a staggered cut at a specific sequence of DNA and results in sticky ends\
			- ![[EcoR1-diagram.png]]
				- diagram shows the cut that [[EcoR1]] makes
				- notice staggered cut at sequence GAATTC
					- only sequence that EcoR1 recognizes
			- named after E. coli strain R1
			- joining of fragments enabled by pairing of complementary nucleotides w/ any end that's formed from an EcoR1 cut
		- [[restriction enzymes]] are often used to insert a gene of interest into a plasmid
			- first, plasmid and gene of interest must be cut w/ same restriction enzyme (ex. EcoR1 in diagram below)
			- cut plasmid and gene of interest then mixed together w/ enzyme [[ligase]] and allowed to come together
			- ![[EcoR1-insertion-into-plasmid.png]]
- ### 7.1.3: Biotechnology: Gene Cloning
	- [[gene cloning]]: process by which multiple copies of a gene are produced
		- ![[gene-cloning.png]]
			- in diagram, bacterial plasmids that has a resistance gene (amp^R) is cut w/ a particular [[restriction enzymes]] (sing.)
				- at cut site is the lacZ gene.
			- DNA, w/ the human insulin gene somewhere in it, is cut w/ the same restriction enzyme
				- cut plasmids and cut DNA mixed tegether w/ [[ligase]], allowing [[sticky ends]] to connect and ligate
			- ![[gene-cloning-result.png]]
				- resulting plasmids put into bacteria by adding chemical such as calcium sulfate, which makes membrane more soluble
				- bacteria screened for presence of plasmid by being grown on a medium with [[ampicillin]]
					- bacteria that have plasmid will be resistant to [[ampicillin]]
				- bacteria then screened for presence of a gene inserted into the plasmid
					- if gene has been inserted into plasmid, then the lacZ gene (located at cut site) won't be functional
					- if no gene has been inserted into plasmid, lacZ will be fnctional
						- if lacZ = functional in plasmid, will cause color change in the medium if certain chemicals are used
						- bacteria that don't cause a color change are the ones that have a gene inserted into the plasmid
					- once bacteria have been screened for the presence of the plasmid and for the presence of a gene insert in the plasmid, the bacteria are then screened for the specific gene 
		- involves cutting genes out of a chromosome and inserting them into [[plasmids]], which can then be placed into bacteria, which will divide, forming multiple copies of gene
		- bacterial will also be able to express gene and form gene product
		- good plasmid for gene cloning must have particular restriction sites + genes that make it easily detectable
		- plasmid and gene of interest must both be cut w/ same restriction enzyme in order to create complimentary [[sticky ends]]
			- joining together of sticky ends allows gene to be easily inserted into the plasmids
		- plasmids w/ DNA insertions are placed into bacteria, which must then be screened for the presence of the gene of interest
		- 
- ### 7.1.4: Biotechnology: Detection of Cell Clones
	- as bacteria containing [[plasmids]] w/ inserted genes multiply, cell [[clones]] are produced
		- clones must be screened in order to determine which clones have plasmids containing the gene of interest
		- ![[screening-for-gene-of-interest.png]]
			- cells being screened for gene of interest must be cultured and then transferred to filter
				- cells must then be denatured to expose the DNA and separate the complementary strands of the DNA
			- DNA on filter then exposed to DNA probe
				- if probe matches sequence on the filter, then it'll hybridize to the DNA by complementary base pairing
				- filter must then be rinsed to wash off all excess unhybridized probe
			- filter then exposed to photographic film, which shows spots where it's been exposed to radioactive probes
				- images on film correspond to cells that contain gene of interest
	- shotgun approach to detecting bacteria for gene of interest = testing each colony for presence of gene or gene product of interest
		-  this method can be used to screen bacteria for presence of particular gene
		- [[DNA probe]]: synthesized segment of radioactively labeled DNA (or RNA) used to find a gene of interest
			- ![[DNA-probe.png]]
				- can be constructed using DNA segments containing [[radioisotope]]s (which give off radiation that can be detected on film)
				- sequence of a portion of the gene of interest must be known to construct probe segment that'll base-pair with a single strand of DNA
			- radioactive DNA probes can be constructed for the detection for specific DNA sequences within cells
			- will hybridize w/ complementary sequence of DNA, allowing it to be distinguished from other DNA sequences
## 7.2: Techniques in Biotechnology
- ### 7.2.1: Biotechnology: Reverse Transcriptase: A Tool Taken from Viruses
	- [[reverse transcriptase]]: enzyme that makes DNA from an RNA template
	- [[retroviruses]]: depend on reverse transcriptase to propagate themselves within a host cell
		- ![[visualizing-retroviruses.png]]
		- exception to the 'central dogma of biology'
		- can make DNA from RNA using reverse transcriptase. examples:
			- HIV
				- ![[HIV-visualization.png]]
				- beginning at top of diagram, [[HIV]] enters cell and uses reverse transcriptase to make a DNA strand from its RNA
				- synthesized DNA strand is then used as a template to make complementary strand of DNA
				- double-stranded DNA segment will then enter the nucleus of the host cell and integrate into the host chromosome
					- in the nucleus, RNA is transcribed from the viral DNA
				- after being translated by cell, viral RNA directs synthesis of viral protein, such as reverse transcriptase, and components of the viral coat
					- new viruses then assembled and bud off of the host cell
			- common cold
	- [[HIV]] (Human Immunodeficiency Virus), the virus that causes [[AIDS]], is a retrovirus that uses reverse transcriptase to multiply itself within a host cell
	- [[reverse transcriptase]] can also be used as a tool in [[biotechnology]]
- ### 7.2.2: Using Reverse Transcriptase to make cDNA
	- **review**: during [[transcription]], a strand of RNA is synthesized from a DNA template
		- in eukaryotes, non-coding regions called [[introns]] must be removed from the RNA strand in order for it to coded for a functional protein
	- as a lab technique, [[reverse transcriptase]] can be used to make DNA from RNA. the DNA that results from an RNA template = [[cDNA]] or "complementary DNA", which is must shorter than the og gene bc it lacks introns
	- in a [[DNA microarray assay]] researchers can test for gene expression in a particular tissue
	- a [[cDNA]] library is a collection of many bacterial cell clones, each containing a gene that was expressed in the tissue from which mRNA was collected
	- ![[Pasted image 20230913161422.png]]
		- gene for the clotting protein, factor VIII, is made up of 186,000 base pairs
		- RNA that it codes for has 9000 base pairs
		- part of the reason that the actual gene has so many more base pairs than the RNA is because of the presence of introns 
	- ![[RVRSEtrnscrpts-in-DNA-from-RNA.png]]
		- Reverse transcriptase can be used to make a DNA segment from RNA (as shown above)
		- The DNA segment produced is called complementary DNA or cDNA 
			- Complementary DNA is much shorter than the original gene because it lacks introns
			- It can be cloned more easily in bacterial cells because they lack the ability to remove introns from RNA transcripts
	- ![[DNA-microarray-assay.png]]
		- In a DNA microarray assay, researchers test for the genes expressed in a particular tissue
			- RNA is first extracted from a particular type of tissue and labeled with a fluorescent dye
			- The labeled RNA segments are added to the DNA microarray (DNA fragments from specific genes fixed to a slide)
			- If the sequences are complementary, the RNA/DNA complex will fluoresce
			- Researchers have thus traced the RNA to the appropriate gene
		- The RNA from the tissue can also be used to make cDNA
			- Each cDNA that is produced from RNA can be placed into a plasmid and put into a bacterium to make a cDNA library
			- The library produced will contain only genes expressed in the particular tissue from which it was constructed
- ### 7.2.3: Electrophoresis: Separating DNA
	- The process of [[electrophoresis]] separates DNA segments based on their rate of movement through a gel under an electric field
		- This rate of movement is related to the size of the DNA segments. 
	- Under an electric current, small molecules will travel through the gel more easily (and therefore faster) than larger molecules. 
	- Electrophoresis provides scientists with a valuable tool for studying molecules such as DNA, RNA, and proteins.
	- ![[electrophoresis.png]]
		- Electrophoresis is a method used to separate DNA segments on the basis of their rate of movement through a gel under an electric field
		- DNA has an overall negative charge
		- A DNA mixture is placed within a well of an electrophoresis gel (as seen in the diagram to the left)
		- The gel is within a box filled with a buffer solution
		- Electrodes on opposite ends of the box form a current, which causes the DNA mixture to travel from the negative electrode (the cathode) down through the gel towards the positive electrode (the anode)
	- ![[DNA-electrophoresis.png]]
		- As the DNA travels down the gel, segments separate based on size
		- Large segments move more slowly through the gel than smaller segments
		- The resulting gel will have bands on it that represent DNA fragments of a particular size
		- Electrophoresis can reveal much about the molecules in a particular mixture
		- The illustration to the left shows the bands resulting from the separation of various plasmids exposed to restriction enzymes
		- Comparing the bands of DNA, we find that the original plasmids must have differed in size
- ### 7.2.4: Sequencing DNA: The Sanger Method
	- The sequence of DNA strands can be determined through use of the Sanger method for DNA sequencing, developed by Frederick Sanger
	- The Sanger method of DNA sequencing involves the use of DNA polymerase to make a strand of DNA that is complementary to the strand of DNA to be sequenced
		- DNA polymerase builds a nucleotide chain by adding new nucleotides to a DNA strand at its terminal 3’ OH group
	- Dideoxyribonucleotides are nucleotides that lack two oxygens. 
		- Once incorporated into a DNA strand, they can block further DNA synthesis because they lack a 3’ OH group
		- Without a 3’ OH group, the DNA polymerase cannot add another nucleotide
	- ![[nucleotide-attachment-by-dna-polymerase.png]]
		- The diagram to the left shows two nucleotides
			- Note both nucleotides have an OH group on the 3’carbon
		- [[DNA polymerase]] is an enzyme that builds a polymer of DNA by adding a nucleotide to the 3’ OH group of another nucleotide or DNA strand
		- The dashed line represents the attachment that will be made between the two nucleotides.
	- ![[inhibiting-dna-strand-extension-with-dideoxynucleotide-ddntp.png]]
		- A dideoxynucleotide (ddNTP) lacks the 3’ OH group
		- When it is incorporated into a DNA strand, the strand cannot be lengthened
		- DNA polymerase will not have the 3’ OH group to add a nucleotide
	- ![[dna-sequencing-by-dideoxy-chain-termination-method.png]]
		- To determine the sequence of a strand of DNA, four separate test tubes are prepared
		- Each test tube contains the following: 
			1. The unknown DNA 
			2. Regular nucleotides: dATP, dTTP, dGTP, and dCTP 
			3. DNA polymerase 
			4. DNA primer (in this case it is ATAA) 
		- In each test tube, one of the four different dideoxynucleotides is also added: ddATP, ddGTP, ddCTP, and ddTTP
			- The complementary strand of DNA is synthesized until a dideoxynucleotide is added
			- The strands that result from each tube are shown in the lower portion of the diagram
	- ![[dna-sequence-determination-by-gel-electrophoresis.png]]
		- When the reaction mixtures have completed polymerization, the DNA strands from each tube are separated by gel electrophoresis
		- In this example, note that the shortest fragment was in the GTP lane of the gel
			- G is the first base in the new DNA strand
			- The next shortest fragment is in the T lane
			- Therefore, T is the second base in the DNA strand.
		- Based on the results in the gel, the sequence following the primer is GTACT.
## 7.3: More Techniques in Biotechnology
- ### 7.3.1: Restriction Fragment Length Polymorphisms: Genetic Markers
	- Restriction Fragment Length Polymorphisms ([[RFLPs]]) are differences in homologous sequences of DNA that can be detected between individuals
	- RFLP analysis involves exposing homologous segments of DNA to a restriction enzyme and separating the resulting fragments by [[electrophoresis]]
	- Since RFLPs result from differences in DNA sequences, they can be passed along to offspring
	- RFLPs that are associated with particular genes can serve as [[genetic markers]] for those genes
	- ![[restriction-fragment-length-polymorphisms.png]]
		- Restriction fragment length polymorphisms ([[RFLPs]]) are small differences in DNA sequences between individuals
		- RFLPs are detected by examining the different restriction fragments that result after exposure to the same [[restriction enzyme]]
		- When two homologous pieces of DNA have different restriction fragments after exposure to restriction enzyme, the pieces are said to be polymorphic.
	- ![[single-base-pair-mutation-in-rflp-patterns.png]]
		- A single base-pair mutation can cause a difference in RFLP patterns
		- In the diagram to the left, DNA piece 1 has two restriction enzyme cut sites
		- Three fragments of DNA will result after the DNA is exposed to the restriction enzyme: fragments w, x, and y
		- As seen in DNA piece 2, the difference in a single base pair causes a restriction enzyme cut site to be lost
			- Now only two fragments of DNA will result after the DNA is exposed to restriction enzyme: fragments y and z
	- ![[rflps-as-genetic-markers-for-disease.png]]
		- RFLPs can be passed to offspring like genes and can be used as [[genetic markers]] for particular genes
		- For example, a particular RFLP could be linked to a disease, such as cystic fibrosis
			- If the RFLP is found in all carriers of cystic fibrosis, then it can be used as a genetic marker for that disease.
- ### 7.3.2: Polymerase Chain Reaction: DNA Amplification
	- Many copies of a DNA segment can be made through the process of denaturing the DNA segment and then polymerizing the new strand using [[DNA polymerase]]
		- In the past this process was difficult because the heat required to denature DNA would also denature DNA polymerase
	- Thomas Brock found that the DNA polymerase of the bacterium T. aquaticus would not denature at high temperatures
	- Building on Brock’s findings, Kary Mullis developed the method of DNA amplification known as [[PCR]] (polymerase chain reaction)
		- This method overcame the problem of denaturing DNA polymerase at high temperatures by using the DNA polymerase of [[T. aquaticus]]
		- In 1993, Mullis was awarded the Nobel Prize in chemistry for his development of PCR
		- PCR is a technique that amplifies a segment of DNA into millions, even billions of copies in a short period of time
	- ![[dna-amplification-using-taq-polymerase.png]]
		- A segment of DNA can be amplified, or made into many copies. First the DNA segment to be amplified must be denatured into single strands. 
			- A primer, [[DNA polymerase]], and all four nucleotides must then be added. DNA polymerase will build a new strand of DNA that is complimentary to the single strand of DNA. 
		- The problem with this method is that the temperature required to denature DNA will also denature DNA polymerase. 
		- A bacteriologist by the name of Thomas Brock studied a bacterium that lives in the hot springs of Yellowstone National Park. 
			- This bacterium, *[[Thermus aquaticus]]*, exists in a high temperature environment and uses a DNA polymerase that can withstand such temperatures. 
				- The DNA polymerase derived from [[T. aquaticus]] is referred to as Taq polymerase.
	- ![[kary-mullis-pcr-method-with-taq-polymerase.png]]
		- Kary Mullis developed the polymerase chain reaction ([[PCR]]), the method for DNA amplification that employs [[Taq polymerase]] and won a Nobel prize in 1993 for his work. 
		- The DNA segment to be amplified is mixed with Taq polymerase, nucleotides, and primer and is then heated up. 
		- The DNA segment denatures and the mixture is allowed to cool. 
		- As it cools, the primer binds to the DNA strand and Taq polymerase synthesizes a strand of DNA complimentary to the single strand. 
			- We have now doubled our DNA
	- ![[multiple-rounds-of-dna-amplification.png]]
		- The doubled DNA strands may then undergo this process again. The DNA is denatured and the reaction mixture cools. 
			- While cooling, the primer attaches to the single strands of DNA and Taq polymerase builds a new strand. 
			- The DNA strands have been doubled again! 
		- If the mixture is heated up and cooled repeatedly, and there are enough primer segments and nucleotides present in the mixture, this process can create millions of DNA segments from a single piece of DNA in a relatively short period of time.

- ### 7.3.3: DNA Fingerprinting
	- Tandem repeats are repeating sequences of DNA (such as the sequence 5’ CATCATCATCAT 3’). 
		- Variable number tandem repeats ([[VNTRs]]) are repeating sequences of DNA that vary in their number of repeating units between individuals. 
	- VNTRs can be used to differentiate between the DNA of individuals in a process known as DNA fingerprinting.
	- In a typical DNA fingerprint, several VNTR markers are compared in order to decrease the odds that more than one person will have identical DNA fingerprints.
	- ![[sample-of-blood-extracted-and-vntr-amplified.png]]
		- A sample of blood is extracted from a glove at a crime scene and placed in a test tube. 
		- A specific variable number tandem repeat ([[VNTR]]) sequence is amplified using the [[polymerase chain reaction]] technique.
	- ![[vntr-amplification-from-suspect-blood-samples.png]]
		- A sample of blood is taken from the two suspects (you and George). The same VNTR that was amplified from blood at the crime scene is amplified in each of the suspect’s samples. 
		- George’s VNTR has 12 repeating units, while your sample has 3 repeating units.
	- ![[gel-electrophoresis-for-vntr-separation-and-matching.png]]
		- Each of the amplified VNTR samples is separated using [[gel electrophoresis]]. Remember: small DNA molecules move more quickly in the gel than large DNA molecules; thus, small molecules will travel farther in the gel within a given period of time. 
		- It looks like George is a match! But George isn’t guilty yet! A single VNTR is not typically sufficient to prove anything because the odds that two people have the same VNTR can be relatively high (for example, one in forty).
	- ![[multiple-vntr-comparison-for-identifying-suspect.png]]
		- Several VNTR’s must be compared in order to reduce the odds that two people will have the same fingerprint. For example, if the odds of two people having the same VNTR were one in forty and remained constant, the odds of two people having six identical VNTRs would be one in 4,096,000,000 (1/40 x 1/40 x 1/40 x 1/40 x 1/40 x 1/40)! 
		- If after several VNTRs have been compared and George’s DNA matches, then the chances are remarkably high that it was George’s blood on the glove. See you later George!
- ### 7.3.4: Southern Blotting
	- [[southern blotting]] is a laboratory technique for DNA visualization that was developed by E. M. Southern in 1975
	- The process of Southern blotting involves the transfer of DNA segments from an [[electrophoresis]] gel to a sheet of nylon paper (or nitrocellulose membrane) and hybridizing the segments with a radioactive probe.
	- By taking an [[autoradiogram]] of the nylon paper, a photographic negative will reveal locations of the bands of DNA that contain the target sequence.
	- Southern blotting is a procedure used in conjunction with many other techniques in biotechnology, such as [[RFLP analysis]] and [[DNA fingerprinting]].
	- ![[visualize-dna-bands-using-southern-blotting.png]]
		- Specific DNA segments within an [[electrophoresis]] gel can be visualized with the [[Southern blotting]] technique.
		- Suppose that you want to find out which of the bands in the gel to the left contain the sequence AAATTT.
	- ![[southern-blotting-process-for-dna-visualization.png]]
		- Southern blotting can be used to determine which of the bands in the gel to the left contain the sequence AAATTT.
		- The gel is placed into a container that has special buffers and chemicals that are able to denature the DNA. The gel is then placed onto a platform that is covered with filter paper within the container. A positively charged piece of membranous paper (nylon or nitrocellulose) is placed on top of the gel. A stack of paper towels is placed on top of the charged paper and weight is applied.
		- As solution is drawn up through the layers of paper, DNA is transferred from the gel to the charged nylon membrane. The DNA within the membrane is denatured by the solution, which results in bands of single stranded DNA positioned as they were in the gel.
	![[autoradiography-for-dna-probe-hybridization.png]]
		- To visualize the specific sequence of DNA, the membrane is placed in a plastic bag containing a [[radioactive DNA probe]]. The probe, with a sequence complimentary to the target sequence (in this case TTTAAA), is allowed to hybridize with the DNA. The membrane is then rinsed to wash off excess unhybridized probe.
		- In a process called [[autoradiography]], the membrane is exposed to film paper. Wherever the radioactive probe hybridizes with DNA containing a complimentary sequence, there will be a corresponding image on the film paper.
- ### 7.3.5: Detecting DNA Homology: A Biotechnology Summary
	- Various techniques are employed in biological research projects
	- Mr. Wolfe’s research determined whether the level of DNA homology between the antennapedia gene in Drosophila melanogaster and the [[antennapedia]] gene in the Nasonia wasp. This project employed the use of a [[plasmid]] from a [[gene library]], [[restriction enzymes]], and [[Southern blotting]].
## 7.4: Human Genome Project
- ### 7.4.1: The Human Gene Pool
- ### 7.4.2: The Human Genome Project: Recent Findings
- ### 7.4.3: The Human Genome Project: Applications